### PR TITLE
feat: add option to disable client-side certificate requirement

### PR DESCRIPTION
**Pull request by gilmaicor**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
* dojot/dojot#2213

* **Other information**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,10 +70,12 @@ These are the environment variables used by iotagent-http
...
 | LOG_LEVEL                    | Set log level                                    | "info"                     |
+| REQUEST_CERT                 | Required client-side certificate                 | "true"                     |
 
 ## Notes
 
 - If a timestamp is not part of the message sent by a device, it will be added by the iot-agent http.
 - Invalid values ​​for "ALLOW_UNSECURED_MODE" will be considered false. To enable set to "true".
 - With HTTP the tenant and device id in the body are mandatory, with HTTPS they are optional, but if both are passed they must match the cname.
 - In the x509-identity-mgmt service be sure to set the X509IDMGMT_CERTIFICATE_CHECK_SUBJECTDN variable to "true", if it is passed the certificate cname will be tenant:device, otherwise the HTTPS request will not work
+- If REQUEST_CERT is false, the client must inform that it is making an unsafe request using the options provided by the tool used (ex: "-k"/"--insecure" will make curl skip certificate validation, it will not turn off SSL all together).
```
Comment (rascaraficci) [inline README.md:81]: Sugestão:

If REQUEST_CERT is false, it will not be required the client certificate. The default value is true.

**Review by rascaraficci — COMMENTED** [review]
Good job. I left a minor comment!

**Review by rascaraficci — APPROVED** [review]
It looks OK to me!

File: example/docker-compose/docker-compose.yml [hunk 2]
```diff
@@ -426,6 +426,7 @@ services:
       - device-manager
     ports:
       - 8080:3124
+      - 8081:3123
```
Comment (luissajunior) [inline example/docker-compose/docker-compose.yml:429]: Why this port is needed?

Comment (gilmaicor) [reply example/docker-compose/docker-compose.yml:429]: To test connection with HTTP.

**Review by luissajunior — APPROVED** [review]

**Review by luissajunior — APPROVED** [review]

**Review by luissajunior — APPROVED** [review]

**Review by luissajunior — APPROVED** [review]

**Review by luissajunior — APPROVED** [review]

**Review by luissajunior — APPROVED** [review]
Ok

**Pull request merged**